### ISSUE TITLE
Episode Rename - "{episode:0}" is marked as invalid yet is explicitly documented

### ISSUE THREAD

**Issue by anaisbetts** [Sonarr/Sonarr#5022]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am trying to set the episode format to `{season:0}x{episode:0}` (i.e. 1x1.mkv, 1x2.mkv, etc), however this is reported as invalid in the UI. `{season:0}x{episode:00}` works correctly.

### Expected Behavior

`{season:0}x{episode:0}` and `{season:0}x{episode:00}` should both work

### Environment

```markdown
- OS: Ubuntu 20.04
- Sonarr: 3.0.8
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: N/A
```

### What branch are you running?

Main

### Trace Logs?

N/A

### Anything else?

N/A
